lib/serializer/CMemorySerializer.cpp

```diff
@@ -32,9 +32,8 @@ int CMemorySerializer::write(const void * data, unsigned size)
 	return size;
 }
 
-CMemorySerializer::CMemorySerializer(): iser(this), oser(this)
+CMemorySerializer::CMemorySerializer(): iser(this), oser(this), readPos(0)
```

Comment:
`readPos` value could be moved to declaration I guess

Reply:
It can, but as discussed in Slack, @IvanSavenko tells me than he prefers this approach - if we have a non-default constructor, initalize all fields in constructor, not in declaration.

What is better?

Reply:
aha, let's keep then

Reply:
I would rather keep it here. 
Not 100% sure what solution in regards to in-place initialization is better, so I think for now we should avoid too many changes. Let's do parts on which everyone agrees. That's more than enough things to do anyway.